presto-hive/src/main/java/io/prestosql/plugin/hive/HiveHdfsModule.java

```diff
@@ -33,6 +33,7 @@ public void configure(Binder binder)
         binder.bind(HdfsEnvironment.class).in(Scopes.SINGLETON);
 
         binder.bind(HdfsConfigurationInitializer.class).in(Scopes.SINGLETON);
+        newSetBinder(binder, ConfigurationInitializer.class);
```

Comment:
How the code worked previously?

Reply:
This fails when  `HiveHdfsModule` is used alone. It works, when is used together with some extension module (like GcsModule).
So this is just making sure modules deps are clean, no external visible change.